hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/TopologySubcommand.java

```diff
@@ -68,6 +69,13 @@
       description = "Print Topology with full node infos")
   private boolean fullInfo;
 
+  @CommandLine.Option(names = {"-n", "--nodeOperationalState"},
```

Comment:
Options are usually preferred in this form: `--operational-state`.  The camelCase form is consistent with the command name, though.